.github/pull_request_template.md

```diff
@@ -31,5 +36,6 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 * Which compiler / HPC you used to run the regression tests in the PR? 
 * Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):    
 Please indicate the expected changes in the outputs ([excluding the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)).
-
+* please add appropriate labels to indicate a change (The labels should be added by the code managemers):    
```

Comment:
Maybe: 
Please list which labels code managers should add to indicate code changes:  _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, ...